app/javascript/Components/assignment_summary_table.jsx

```diff
@@ -30,16 +48,145 @@ export class AssignmentSummaryTable extends React.Component {
     this.fetchData();
   }
 
-  toggleShowInactiveGroups = showInactiveGroups => {
-    let filtered = this.state.filtered.filter(group => {
-      group.id !== "inactive";
+  getColumns = () => {
+    const columnHelper = createColumnHelper();
+
+    const expanderColumn = columnHelper.display({
```

Comment:
Overall, this new column definition is good but should be available more generally, not just in this file. Please (1) move it into the `table.jsx` file and (2) modify the defintion of `Table` itself so that if a subcomponent renderer prop is passed in (i.e., not undefined), then this column is _automatically_ prepended to the columns passed in as props. This will generalize your work to all tables with subcomponents.